clients/drcachesim/common/options.h

```diff
@@ -57,6 +57,7 @@
 // Constants used by specific tools.
```

Comment:
Please augment the PR description to say that this subclassing of tlb_t to add replacement policies is a short-term solution and that i#7287 covers refactoring the replacement policies for a cleaner solution.